log_bucket/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "bucket" {
+  value = "${aws_s3_bucket.logs.bucket_domain_name}"
+}
+
+output "bucket_arn" {
+  value = "${aws_s3_bucket.logs.arn}"
+}
+
+output "bucket_id" {
+  value = "${aws_s3_bucket.logs.id}"
+}
+
+output "bucket_domain_name" {
+  value = "${aws_s3_bucket.logs.bucket_domain_name}"
+}
+
+output "s3_logs_path" {
```

Comment:
I'd suggest not passing out the vars as the same. Since the caller passed them in, it already has these values. the more important ones are the stuff you had above where we are passing out "internal' values